flink-contrib/flink-storm-compatibility/flink-storm-compatibility-examples/src/main/java/org/apache/flink/stormcompatibility/singlejoin/SingleJoinTopology.java

```diff
@@ -1,90 +0,0 @@
-/*
```

Comment:
Okay. I just didn't want a half-ready example to linger around. I will
remove that commit.

2015-06-22 11:58 GMT+02:00 Matthias J. Sax <notifications@github.com>:

> In
> flink-contrib/flink-storm-compatibility/flink-storm-compatibility-examples/src/main/java/org/apache/flink/stormcompatibility/singlejoin/SingleJoinTopology.java
> <https://github.com/apache/flink/pull/853#discussion_r32919318>:
>
> > @@ -1,90 +0,0 @@
> > -/*
>
> Why do you remove it? It's going to be fixed soon.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/flink/pull/853/files#r32919318>.
>